### PR TITLE
W2 - hieucap - create drawerlayout screen

### PR DESCRIPTION
**Pull request by at-hieucap**
#### Related issues (Optional)
Task #29 

#### Description (Required)
- [x] Create screen

### CODE REVIEW

File: app/src/at-hieucap/java/vn/asiantech/internship/ui/drawer/layout/DrawerLayoutActivity.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        if (requestCode == REQUEST_CODE) {
+            if (data != null) {
+                try {
+                    Uri chosenImageUri = data.getData();
+                    Bitmap bm = MediaStore.Images.Media.getBitmap(getContentResolver(), chosenImageUri);
+                    Drawable d = new BitmapDrawable(getResources(), bm);
+                    ((User) mInformationList.get(0)).setAvatar(d);
+                    mAdapter.notifyItemChanged(0);
+                } catch (FileNotFoundException e) {
+                    e.printStackTrace();
```
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/ui/drawer/layout/DrawerLayoutActivity.java]: WARNING: [AvoidPrintStackTrace] Avoid printStackTrace(); use a logger call instead.
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/logging-java.html#AvoidPrintStackTrace

File: app/src/at-hieucap/java/vn/asiantech/internship/ui/drawer/layout/DrawerLayoutActivity.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                } catch (IOException e) {
+                    e.printStackTrace();
```
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/ui/drawer/layout/DrawerLayoutActivity.java]: WARNING: [AvoidPrintStackTrace] Avoid printStackTrace(); use a logger call instead.
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/logging-java.html#AvoidPrintStackTrace

File: app/src/at-hieucap/java/vn/asiantech/internship/ui/drawer/layout/Option.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package vn.asiantech.internship.ui.drawer.layout;
+
+public class Option {
+    public int icon;
+    public String name;
+    private boolean isClicked;
```
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/ui/drawer/layout/Option.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isClicked has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

**Comment by at-hieucap** [comment]
@at-vinhhuynh  , I fix finish , please review for me
Thank you so much

**Comment by at-hieucap** [comment]
@at-huongnguyen I fix finished , Please review for me,
Thank you so much!!!

**Comment by at-huongnguyen** [comment]
@at-hieucap 
Plz resolve conflict.

**Comment by at-hieucap** [comment]
@at-huongnguyen I fix finished , Please review for me,
Thank you so much !

File: app/src/at-hieucap/java/vn/asiantech/internship/models/Option.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
+package vn.asiantech.internship.models;
+
+import android.content.Intent;
+
+public class Option {
+    public int icon;
+    public String name;
+    private boolean isClick;
```
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/models/Option.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isClick has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

File: app/src/at-hieucap/java/vn/asiantech/internship/models/Option.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
+package vn.asiantech.internship.models;
+
+import android.content.Intent;
+
+public class Option {
+    public int icon;
+    public String name;
+    private boolean isSelected;
```
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/models/Option.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isSelected has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

File: app/src/at-hieucap/java/vn/asiantech/internship/ui/drawerlayout/DrawerLayoutActivity.java [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+                try {
+                    Uri chosenImageUri = data.getData();
+                    Bitmap bm = MediaStore.Images.Media.getBitmap(getContentResolver(), chosenImageUri);
+                    Drawable d = new BitmapDrawable(getResources(), bm);
+                    ((User) mInformationList.get(0)).setAvatar(d);
+                    mAdapter.notifyItemChanged(0);
+
+                } catch (IOException e) {
+                    e.printStackTrace();
```
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/ui/drawerlayout/DrawerLayoutActivity.java]: WARNING: [AvoidPrintStackTrace] Avoid printStackTrace(); use a logger call instead.
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/logging-java.html#AvoidPrintStackTrace

File: app/src/at-hieucap/java/vn/asiantech/internship/models/Option.java (hunk 5 above)
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/models/Option.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isSelected has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

File: app/src/at-hieucap/java/vn/asiantech/internship/ui/drawerlayout/DrawerLayoutActivity.java (hunk 6 above)
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/ui/drawerlayout/DrawerLayoutActivity.java]: WARNING: [AvoidPrintStackTrace] Avoid printStackTrace(); use a logger call instead.
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/logging-java.html#AvoidPrintStackTrace

File: app/src/at-hieucap/java/vn/asiantech/internship/ui/drawerlayout/DrawerLayoutActivity.java [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+            if (data != null) {
+                try {
+                    Uri chosenImageUri = data.getData();
+                    Bitmap bm = MediaStore.Images.Media.getBitmap(getContentResolver(), chosenImageUri);
+                    Drawable d = new BitmapDrawable(getResources(), bm);
+                    ((User) mInformationList.get(0)).setAvatar(d);
+                    mAdapter.notifyItemChanged(0);
+                } catch (IOException e) {
+                    Log.e(TAG, MSG);
+                    e.printStackTrace();
```
Comment (at-dev-support-01) [inline app/src/at-hieucap/java/vn/asiantech/internship/ui/drawerlayout/DrawerLayoutActivity.java]: WARNING: [AvoidPrintStackTrace] Avoid printStackTrace(); use a logger call instead.
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/logging-java.html#AvoidPrintStackTrace

**Comment by at-hieucap** [comment]
@at-huongnguyen I fixed comments , Please review for me,
Thank you so much !

**Pull request merged**